virttest/nfs.py

```diff
@@ -332,6 +332,32 @@ def setup_remote(self):
                 raise exceptions.TestFail("Failed to run %s: %s" % (mkdir_cmd, o))
             self.mkdir_mount_remote = True
 
+        # Check firewall in host permits nfs service to mount nfs directory
+        # from remote machine
+        try:
+            firewall_cmd = "firewall-cmd --list-all | grep services:"
```

Comment:
@will-Do I am not sure, but when I tried in RHEL73, nfs mounting was failing as firewall blocked the request from remote machine and dependent tests got failed.